README.md

```diff
@@ -1,70 +1,58 @@
-# Getting Started with Create React App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+#  MICROVERSE CAPSTONE 3 
 
-## Available Scripts
+> This project was built using react redux to fetch 
+> covid data for each country.
 
-In the project directory, you can run:
+Home view                                      |  Information view
+:------------------------------------------------:|:-----------------------------------------------:
+![](./src/capstone.PNG)                     |  ![](./src/capsone2.PNG)
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Built With
+- Css
+- React 
+- React redux
+- React test libraries
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Link 
+[LINK Live](https://quirky-austin-ee8614.netlify.app/)
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Vidoe Description
 
-### `npm run build`
+[Video Demo](https://www.loom.com/share/c1bafc0ccb0d4ed4995be48480abd796)
```

Comment:
- [ ] In the video you are expected to talk about a piece of code that you found interesting. Please record an additional video highlighting this.